### PR TITLE
fix: ds-controller-board unit test

### PR DESCRIPTION
**Pull request by vli11**
Signed-off-by: Valina Li <valina.li@intel.com>

### CODE REVIEW

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: ds-controller-board/device/virtual_test.go [hunk 1]
```diff
@@ -10,7 +10,7 @@ import (
 	"testing"
 
 	"github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
-
+	dsModels "github.com/edgexfoundry/device-sdk-go/v2/pkg/models"
```
Comment (lenny-intel) [inline ds-controller-board/device/virtual_test.go:13]: This is the same package as above, just aliased. Remove this one and use the non-aliased one.

File: ds-controller-board/driver/driver_test.go [hunk 2]
```diff
@@ -174,3 +160,144 @@ func CreateControllerBoardDriver(t *testing.T, virtual bool, initialize bool, ex
...
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			drv := &ControllerBoardDriver{
+				lc:     tt.lc,
+				config: tt.config,
+			}
+			if err := drv.Initialize(tt.lc, make(chan *dsModels.AsyncValues), make(chan<- []dsModels.DiscoveredDevice)); (err != nil) != tt.wantErr {
+				t.Errorf("ControllerBoardDriver.Initialize() error = %v, wantErr %v", err, tt.wantErr)
+			}
```
Comment (lenny-intel) [inline ds-controller-board/driver/driver_test.go:210]: Too much going on in one line for readablity. Please break it up like we did for your test in other PR and use require.Error/require.NoError

File: ds-controller-board/driver/driver_test.go [hunk 3]
```diff
@@ -174,3 +160,144 @@ func CreateControllerBoardDriver(t *testing.T, virtual bool, initialize bool, ex
...
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			drv := CreateControllerBoardDriver(t, true, true, tt.expectedValue)
+			drv.controllerBoard = tt.controllerBoard
+			actual, err := drv.HandleReadCommands(tt.deviceName, tt.protocols, []dsModels.CommandRequest{tt.reqs})
+			if tt.wantErr {
+				require.Error(err)
+				require.Nil(actual)
```
Comment (lenny-intel) [inline ds-controller-board/driver/driver_test.go:290]: add `return` here to avoid else

**Review by lenny-intel — CHANGES_REQUESTED** [review]

**Review by lenny-intel — APPROVED** [review]
LGTM

**Review by joshua-silverio — APPROVED** [review]
LGTM

File: ds-controller-board/driver/driver.go [hunk 4]
```diff
@@ -131,7 +133,7 @@ func (drv *ControllerBoardDriver) HandleWriteCommands(deviceName string, protoco
 				return err
 			}
 		default:
-			return fmt.Errorf("unknown Command Type: '%d'", cmdType)
+			return fmt.Errorf("unknown Command Type: '%v'", cmdType)
```
Comment (brian-intel) [inline ds-controller-board/driver/driver.go:136]: Good change. We don't know what this type will be so allowing go to do the casting is much better

**Review by brian-intel — APPROVED** [review]

**Review by marcpfuller — APPROVED** [review]
LGTM

**Pull request merged**